### DISCUSSION TITLE
Meldung in HA: (Reparaturproblem) Das Add-on RaspberryMatic CCU wurde aus dem Repository entfernt, aus dem es installiert wurde ...

### DISCUSSION THREAD

**Discussion by CjmHo** [OpenCCU/OpenCCU#2798] [🇩🇪 Fragen&Antworten] [answered]
Hallo, 

vielleicht hat jemand diesen Effekt schon gesehen und kann mir helfen:

Ich nutze RaspberryMatic als AddOn in Home Assistant und bekomme regelmäßig das im Titel genannte Reparaturproblem angezeigt.

Ich bin mir nicht ganz sicher, ob es damit etwas zu tun hat, aber es ist mir  zum ersten mal aufgefallen,  nachdem ich das Add-on nochmal runterschmeißen und neu installieren musste, um das Problem mit der eingebetteten WebUI zu beheben, das ich nach dem Upgrade auf 3.75.7 hatte. Kann es sein, dass dabei etwas schief gelaufen ist?       

Bin für jeden Hinweis dankbar.

**Comment by davidkoch88** [comment]
Ich habe bei mir auch seit einigen Wochen das Problem 😔

**Reply by xasz** [reply]
Jap Problem ist bei mir jetzt auch da :(

**Comment by Baxxy13** [comment, marked as answer]
Hab ich bei meinen 3 HA (Test-) Systemen noch nie gesehen.
Habt ihr mal versucht das Repository zu entfernen und neu hinzuzufügen?

**Reply by davidkoch88** [reply]
Leider kann man das Repositiry nicht löschen, da ein add aus ihm installiert ist. Ich müsste erst das add on deinstallieren. 🙈

**Reply by jens-maus** [reply]
Ist doch auch kein problem: Ein Backup innerhalb von RaspberryMatic machen, Addon löschen, Repository löschen, Addon neu installieren und dann das Backup wieder einspielen.. fertig.

**Reply by davidkoch88** [reply]
Backup erstellen und neu einspielen nach reinstallation vom Raspberrymatic funktioniert 😉👍

**Reply by xasz** [reply]
Werd ich wohl dann irgendwann ausprobieren, danke für den Tipp.

**Reply by CjmHo** [reply]
Hatte am Wochenende endlich auch die Zeit, Addon und Repository runterzuschmeißen und dann neu zu installieren. (Wie gut, dass man im RaspberryMatic die Konfigurationen so bequem sichern und wiederherstellen kann 😉 ).
Seitdem wird die Meldung nicht mehr angezeigt. Problem gelöst!  👍
Danke an alle, die sich an der Diskussion beteiligt haben.

**Comment by dkoch-vector** [comment]
Leider ist der Fehler bei mir nun das zweite mal aufgetaucht.

**Reply by xasz** [reply]
Bei mir auch weiterhin als Meldung vorhanden. Habe leider keine Lösung gefunden.

**Reply by maudani** [reply]
Dieses Problem tritt bei mir in den letzten Monaten ständig auf. Zuerst habe ich wie vorgeschlagen, ein Backup gezogen, das AddOn deinstalliert und neu installiert. Nach den 4. Mal war mir das zu nervig; seitdem ignoriere ich die Meldung. Ich bin auch an einer nachhaltigen Lösung interessiert.

**Reply by dkoch-vector** [reply]
Hallo zusammen, ich habe mittlerweile Raspberrymatic nativ auf dem RPI3 laufen und HomeAssist auf einen alten Laptop ausgelagert. Der Laptop hat genügend Power und verbraucht trotzdem im Idle nur 5-7W. Ich denke der RPI3 ist vielleicht generell ein wenig schwach für Home-Assist im Produktiven Einsatz.
Seitdem keine Probleme mehr.

**Comment by tzdisco4** [comment]
Moin zusammen, ich habe diese Meldung auch seit heute morgen. Stellt sich mir die Frage, warum HA denkt, Raspberrymatic sei aus dem Repo entfernt worden. Nach dem Prinzip "keine Wirkung ohne Ursache" sollte doch eine zu finden sein. Der Supervisor hat es bei mir vor 12 h gemeldet. Wurde da was am Repo geändert?

**Reply by xasz** [reply]
Soweit ich es verstehe ist es genau andersrum. hA erwartet seit ein paar Versionen eine Änderung am Repositiry des Addons. Da das Addons Repository nun wohl nicht aktualisiert wurde, wird die Version nicht mehr korrekt erkannt oder so.

**Reply by tzdisco4** [reply]
Moin, danke für den Hinweis. Seltsam finde ich, dass ich Raspberrymatic erst vor ganz kurzer Zeit als HA Addon installiert habe. Das hat problemlos funtioniert. Da die ersten Meldungen hier vom Juli letzten Jahres sind, muss das Problem wohl nur unter bestimmten Bedingungen auftreten. Vielleicht finde ich ja Hinweise bei HA dazu. Schaun mer mal.

**Reply by tzdisco4** [reply]
Zur Info: Ich habe die Meldung einfach ignoriert und inzwischen ist sie nicht mehr vorhanden.

**Reply by Stephan1965** [reply]
Bei mir ist diese Meldung jetzt auch seit einigen Tagen da. Zuerst habe ich ok gedrückt und dann wurde so nach und nach Homatic nicht mehr funktionionsfähig. Dann habe ich ein Backup von HA eingespielt und jetzt ignoriere ich die Meldung. Mal sehen, ob sie auch bei mir von alleine wieder  verschwindet.

**Reply by oradke** [reply]
Wechsele mal auf die aktuelle Version von OpenCCU, das könnte das Problem lösen. Durch die Umbenennugn des Projektts kommt HA manchmal durcheinander.

**Reply by jens-maus** [reply]
Am besten dazu die Anleitung für das Ersetzen das RaspberryMatic HA-Addons zum OpenCCU HA-Addon wie im Folgenden dargestellt befolgen:

https://homematic-forum.de/forum/viewtopic.php?f=65&t=86667

**Reply by Stephan1965** [reply]
OpenCCU läuft, RaspberryMatic ist deinstalliert (in Home Assistant).
Hat bestens funktioniert. Vielen Dank!

**Reply by CFenner** [reply]
Bei mir ist auch erst jetzt der repair issue aufgepopt (2025.11.0b0). Nach dem Backup **in RaspberryMatic** und anschließendes einspielen in openCCU musste ich noch die Container Adresse in der HomematicIP integration anpassen.
